config/application.rb

```diff
@@ -1,16 +1,15 @@
 require_relative 'boot'
-
 require 'rails/all'
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
+# config.assets.initialize_on_precompile = false
```

Comment:
remove it if you are not going to use it

Reply:
Fixed